src/sql/parts/registeredServer/viewlet/serverTreeView.ts

```diff
@@ -428,6 +429,42 @@ export class ServerTreeView {
 		return this._tree.isDOMFocused();
 	}
 
+	/**
+	 * Expand the given element in the tree
+	 */
+	public expand(element: TreeNode | ConnectionProfile): Thenable<void> {
+		return this._tree.expand(element);
+	}
+
+	/**
+	 * Collapse the given element in the tree
+	 */
+	public collapse(element: TreeNode | ConnectionProfile): Thenable<void> {
+		return this._tree.collapse(element);
+	}
+
+	/**
+	 * Reveal the given element in the tree
+	 */
+	public reveal(element: TreeNode | ConnectionProfile): Thenable<void> {
+		return this._tree.reveal(element);
+	}
+
+	/**
+	 * Select the given element in the tree and clear any other selections
+	 */
+	public select(element: TreeNode | ConnectionProfile): void {
+		this._tree.clearSelection();
+		this._tree.select(element);
```

Comment:
We can have multi-select now, correct? If so can we ensure this method aligns with multi-select patterns inside of VSCode APIs?